lms/djangoapps/teams/static/teams/js/views/topic_teams.js

```diff
@@ -47,8 +47,10 @@
 
                 searchTeams: function (event) {
                     event.preventDefault();
-                    // TODO! Will navigate to correct place once required functionality is available
-                    Backbone.history.navigate('browse', {trigger: true});
+                    $('.page-header-search .search-field').focus();
```

Comment:
This didn't work for me when I clicked on the text at the end of the list of teams.